website/blog/2023-11-30-automated-security-versus-the-security-mindset/item.md

```diff
@@ -0,0 +1,163 @@
+
+## The human factor in cybersecurity will always be required
+
+The fact that there are automated tasks and processes in cybersecurity does not mean that the good security as a whole is autonomous or automatic. Security is more about developing a _security mindset_ than a set of features.
+
+For further information on the human element in SaaS security, see [Securing the future of SaaS: Enterprise Security and Single Sign-On](https://goauthentik.io/blog/2023-07-28-securing-the-future-of-saas#good-security-cant-be-automated-the-human-element-in-saas-security).
+
+### Human cyber risk
+
+Humans are at the forefront of cybercrime. Cyber crimes are committed by human beings using adaptation and innovation to invent fresh attack tactics. It is the human mind that continually develops new techniques to hack, infiltrate, and bypass security systems.
+
+For example, if your company does not have a 2FA/MFA credential policy, vulnerabilities exist around whether your staff share user credentials to save them time and stress. If these credentials are not updated regularly, or worse, if they’re shared by email, any moderately skilled, malicious hacker could attempt to access the email account of a single user,  and use it to find other company passwords. It is these human weaknesses and errors that most bad actors hackers rely on.
+
+_Over 80% of malicious hacks are as a result of the exploitation of the widest weakness of all—predictable human behavior._
+
+### Human elements of cybersecurity
+
+Even in a cybersecurity system that is maximally automated, there is human input that can never be removed. Obviously, human experts are needed to guide the automated systems in their functioning. Automation technology depends on humans to set rules and workflows, monitor results over time, and rapidly prioritize then respond to alarming findings.
+
+Once new and significant threats are detached by the automated security, it is human experts again who have to adjust the performance of the automated system as a response to this changing environment. Any further changes need humans to evaluate the performance of automated systems in real-time. Finally, it is humans who train staff in cyber threat detection for these new dangers.
+
+### Human-centered cybersecurity
+
+Despite the growing technology around automated security, and the temptation to relax when it is deployed, there are human factors that are irreplaceable in the practice of cybersecurity. We recently wrote about the importance of the “Blue Team” and how [organizational and product hardening](https://goauthentik.io/blog/2023-11-22-how-we-saved-over-100k#hardening) are an integral part of our human-centered security mindset.
+
+-   The human ability to think creatively and rapidly adapt to changing situations are invaluable to good security processes.
+-   The higher the security risk, the more related processes must be supervised by skilled security professionals.
```

Comment:
This is a humans-first essay, so, uh, let's put the humans first: "The higher the security risk, the more you need skilled security professionals to supervise the security process."